src/Kestrel.Core/Http2Limits.cs

```diff
@@ -74,5 +75,26 @@ public int MaxFrameSize
                 _maxFrameSize = value;
             }
         }
+
+        /// <summary>
+        /// Indicates how much request body data the server is willing to receive and buffer at a time per stream and per connection.
```

Comment:
Added a seperate connection window setting. Cross checking against the stream window is problematic. You don't want to fail the setter based on the value of another property, that's too order dependent. Nor do we have any delayed validation of the server options that I see, this would be a new failure code path.